addons/html_editor/static/src/others/embedded_components/core/file/readonly_file.js

```diff
@@ -32,15 +32,11 @@ export class ReadonlyEmbeddedFileComponent extends Component {
     }
 
     /**
-     * Callback function called when the user clicks on the "Download" button.
-     * The function will simply open a link that will trigger the download of
+     * This function will simply open a link that will trigger the download of
      * the associated file. If the url is not valid, the function will display
      * an error message.
-     * @param {Event} ev
      */
-    async onClickDownload(ev) {
-        ev.preventDefault();
-        ev.stopPropagation();
+    async download() {
```

Comment:
Stable retrocompatibility: Put back the old function name which is a redirection to this one, probably with a comment explaining it's just there for the stable policy.